feature-libs/product-configurator/rulebased/components/attribute/types/radio-button/configurator-attribute-radio-button.component.html

```diff
@@ -1,13 +1,13 @@
 <fieldset>
   <legend style="display: none">{{ attribute.label }}</legend>
   <div id="{{ createAttributeIdForConfigurator(attribute) }}">
-    <div class="form-quantity">
+    <div class="form-quantity" *ngIf="withQuantity">
       <label>{{ 'configurator.attribute.quantity' | cxTranslate }}</label>
       <cx-item-counter
-        [allowZero]="!attribute.selectedSingleValue"
+        [allowZero]="true"
```

Comment:
are we explicitly setting this to `true`? or is it a mistake

Reply:
yes, they should all be able to minus to 0. when quantity equal 0 item is removed from selection